neurovault/apps/statmaps/views.py

```diff
@@ -1202,16 +1202,17 @@ def get_initial_queryset(self):
 
 
 def download_collection(request, cid):
-    from django.http import StreamingHttpResponse
-
     # Files (local path) to put in the .zip
     collection = get_collection(cid, request)
-    filenames = [img.file.path for img in collection.basecollectionitem_set.all()]
+
+    filenames = [img.zip_file.path for img in collection.basecollectionitem_set.instance_of(NIDMResults)] + \
+                [img.file.path for img in
+                 (collection.basecollectionitem_set.instance_of(Image).exclude().instance_of(NIDMResultStatisticMap))]
```

Comment:
It was working in that way in my local, not sure why... Anyway, I corrected it, thanks!
About the test, I was trying many ways of doing it, but I couldn't achieve it, I need help. 

So, I have this `response`, which has a field called `response.streaming_content`, an iterable with the data. I'm reading it as:

```
import zipfile
with io.StringIO() as fl:
    for chunk in response.streaming_content:
        fl.write(chunk)
    zf = zipfile.ZipFile(fl)
```
and I'm getting a `BadZipfile: File is not a zip file` error. I've tried many different solutions, any idea?